### ISSUE TITLE
[NBug] Does not show commits and exception when opening "Open Pull Dialog"

### ISSUE THREAD

**Issue by vsc55** [gitextensions/gitextensions#10457]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.11
- DPI 96dpi (no scaling)

### Issue description

Exit code: 128
Command: C:\Program Files\Git\bin\git.exe
Arguments: remote
Working directory: \\pbxdev....\freepbx\framework\

```
GitExtUtils.ExternalOperationException
 ---> System.Exception
   --- End of inner exception stack trace ---
   at GitCommands.ExecutableExtensions.ExecuteAsync(IExecutable executable, ArgumentString arguments, Action`1 writeInput, Encoding outputEncoding, CommandCache cache, Boolean stripAnsiEscapeCodes, Boolean throwOnErrorExit, CancellationToken cancellationToken)
   at Microsoft.VisualStudio.Threading.JoinableTask.CompleteOnCurrentThread()
   at Microsoft.VisualStudio.Threading.JoinableTask`1.CompleteOnCurrentThread()
   at Microsoft.VisualStudio.Threading.JoinableTaskFactory.Run[T](Func`1 asyncMethod, JoinableTaskCreationOptions creationOptions)
   at Microsoft.VisualStudio.Threading.JoinableTaskFactory.Run[T](Func`1 asyncMethod)
   at GitCommands.ExecutableExtensions.Execute(IExecutable executable, ArgumentString arguments, Action`1 writeInput, Encoding outputEncoding, CommandCache cache, Boolean stripAnsiEscapeCodes, Boolean throwOnErrorExit, CancellationToken cancellationToken)
   at GitCommands.GitModule.GetRemoteNames()
   at GitCommands.Remotes.ConfigFileRemoteSettingsManager.<>c__DisplayClass15_0.<PopulateRemotes>b__0()
   at GitCommands.Remotes.ConfigFileRemoteSettingsManager.PopulateRemotes(List`1 allRemotes, Boolean enabled)
   at GitCommands.Remotes.ConfigFileRemoteSettingsManager.LoadRemotes(Boolean loadDisabled)
   at GitUI.CommandsDialogs.FormPull.BindRemotesDropDown(String selectedRemoteName)
   at GitUI.CommandsDialogs.FormPull..ctor(GitUICommands commands, String defaultRemoteBranch, String defaultRemote, PullAction pullAction)
   at GitUI.GitUICommands.<>c__DisplayClass95_0.<StartPullDialogInternal>g__Action|0()
   at GitUI.GitUICommands.DoActionOnRepo(IWin32Window owner, Func`1 action, Boolean requiresValidWorkingDir, Boolean changesRepo, EventHandler`1 preEvent, EventHandler`1 postEvent)
   at GitUI.GitUICommands.StartPullDialogInternal(IWin32Window owner, Boolean pullOnShow, Boolean& pullCompleted, String remoteBranch, String remote, PullAction pullAction)
   at GitUI.GitUICommands.StartPullDialog(IWin32Window owner, String remoteBranch, String remote, PullAction pullAction)
   at GitUI.CommandsDialogs.FormBrowse.DoPull(PullAction pullAction, Boolean isSilent)
   at GitUI.CommandsDialogs.FormBrowse.ToolStripButtonPullClick(Object sender, EventArgs e)
   at System.Windows.Forms.ToolStripSplitButton.OnButtonClick(EventArgs e)
   at System.Windows.Forms.ToolStripSplitButton.OnMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at GitUI.ToolStripEx.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

I have upgraded from version 3.5 to 4.0 of Git Extensions.
I have also updated to git 2.38.1 as Git Extension requested it.
Now when exploring the repository the commits do not appear and clicking the "Open Pull Dialog" button produces this error.

### Steps to reproduce

1. Open repository explorer.
![imagen](https://user-images.githubusercontent.com/13438676/204129034-89a977ac-8ddb-4f0b-979b-1364cd82d745.png)
3. Click button "Open Pull Dialog".

### Did this work in previous version of GitExtensions?

Until I have updated GitExtension and Git was working fine.

**Comment by RussKie** [comment]
Duplicate of #9954

**Comment by pmiossec** [comment]
Run a git command (like `git ls-files -z --unmerged`) in the command line and git will told you what is the problem and how to fix it. After that GitExtensions will work fine.

**Comment by vsc55** [comment]
Hi @pmiossec,
I have tried and it only fails in repositories accessing via network "\\server\folder".
If I open a local repository "C:\sourcer\" it works fine.

**Comment by pmiossec** [comment]
Output of the git command, please?

**Comment by mstv** [comment]
> I have tried and it only fails in repositories accessing via network "\server\folder".
> If I open a local repository "C:\sourcer" it works fine.

Yes, of course, as expected. So please:

> Run a git command (like `git ls-files -z --unmerged`) in the command line and **git will tell you what is the problem and how to fix it**. After that GitExtensions will work fine.

**Comment by vsc55** [comment]
> Output of the git command, please?

Hi @pmiossec,
Sorry but I don't usually use git via commands.
How can I execute the command referring to a network resource \\server\resource\project"?

**Comment by vsc55** [comment]
Hello again @pmiossec,

I have already solved it with the info of [#9954]
In the end, as simple as adding the following code in ```.gitconfig```. 🤦‍♂️
```
[safe]
	directory = *
```
Thanks and regards.

**Comment by pmiossec** [comment]
Maybe something like that could help https://www.addictivetips.com/windows-tips/access-network-location-in-command-prompt-on-windows-10/
